inventories/README.adoc

```diff
@@ -1 +1,159 @@
+There can be more than one SSOT, each for a different piece of information, but they shouldn't overlap and even less conflict.
+As you create your inventory, you identify these SSOTs and combine them into one inventory using dynamic inventory sources (we'll see how later on).
+Only the aspects which are not already provided by other sources are kept statically in your inventory.
+Doing this, your inventory becomes another source of truth, but only for the data it holds statically, because there is no other place to keep it.
+
+Rationale::
+You limit your effort to maintain your inventory to its absolute minimum and you avoid generating potentially conflicting information with the rest of your IT.
+
+Examples::
+You can typically identify three kinds of candidates as SSOTs:
++
+* technical ones, where your managed devices live anyway, like a cloud or virtual manager (OpenStack, RHV, Public Cloud API, ...) or management systems (Satellite, monitoring systems, ...). Those sources provide you with technical information like IP addresses, OS type, etc.
+* managed ones, like a Configuration Management Database (CMDB), where your IT anyway manages a lot of information of use in an inventory. A CMDB provides you with more organizational information, like owner or location, but also with "to-be" technical information.
+* the inventory itself, only for the data which doesn't exist anywhere else.
++
+Ansible provides a lot of https://docs.ansible.com/ansible/latest/plugins/inventory.html[inventory plugins] to pull data from those sources and they can be combined into one big inventory.
+This gives you a complete model of the environment to be automated, with limited effort to maintain it, and no confusion about where to modify it to get the result you need.
+====
+
+
+.[big]#**Differentiate clearly between "As-Is" and "To-Be" information**#
+[%collapsible]
+====
+Explanations::
+As you combine multiple sources, some will represent:
++
+* discovered information grabbed from the existing environment, this is the "As-Is" information.
+* managed information entered in a tool, expressing the state to be reached, hence the "To-Be" information.
++
+In general, the focus of an inventory is on the managed information because it represents the desired state you want to reach with your automation. This said, some discovered information is required for the automation to work.
+
+Rationale::
+Mixing up these two kind of information can lead to your automation taking the wrong course of action by thinking that the current situation is aligned with the desired state.
+That can make your automation go awry and your automation engineers confused.
+There is a reason why Ansible makes the difference between "facts" (As-Is) and "variables" (To-Be), and so should you.
+Automation is at the end to make sure that the As-Is situation complies to the To-Be description.
```

Comment:
How about:

```suggestion
In the end, automation is making sure that the As-Is situation complies to the To-Be description.
```

Reply:
Your English is generally better than mine. Done.